tests/acceptance/gcp-auth.bats

```diff
@@ -55,6 +55,9 @@ teardown(){
         return
     fi
 
+    # Remove temp credentials file
+    rm ./creds.json
```

Comment:
`teardown` runs after each test. So I am not sure if this will work unless we also `cp` the creds in the `setup` function.

Reply:
Moved `cp` to setup and `rm` to teardown. Thanks!